tests/framework/Optimizers/GeneticAlgorithms/discrete/unconstrained/testGAMaxwoRepTwoPointsCrossover.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers/GA.MaxwoReplacement</name>
+    <author>MohammadAbdo</author>
+    <created>2020-05-16</created>
+    <classesTested>GeneticAlgorithm</classesTested>
+    <description>
+      This test assesses the Genetic algorithm using the weighted sum found in myLocalSum.py function.
+      The nominal dimensionality of the test problem is 3.
+      The objective variable is ans. The problem in unconstrained, it is a maximization problem, and the sampling is from discrete variables without replacement.
+      The cross over mechanism used is the twoPointsCrossover algorithm
```

Comment:
Could you update the description to reflect the fix proposed in this PR?

Reply:
The fix in this PR is to make TwoPointsCrossover work, as it wasn't working before, and this ios the first test on it that's why the last line is added. Can you suggest what should I update the description to?

Reply:
I'm looking for the explanation about failure, and the modifications for the fix @Jimmy-INL